git_machete/client.py

```diff
@@ -934,7 +934,7 @@ def fp_hash(branch_: LocalBranchShortName) -> Optional[FullCommitHash]:
         hook_path = self.__git.get_hook_path("machete-status-branch")
         hook_executable = self.__git.check_hook_executable(hook_path)
 
-        indent_before_branch_name = ' ' * self.__git.get_indent_len_before_branch_name()
+        indent_before_branch_name = ' ' if self.__git.get_boolean_config_attr('machete.status.extraSpaceBeforeBranchName') else ''
```

Comment:
Oh! you've changed it already, gr8

Reply:
I'd call it not `indent` but just
```suggestion
        maybe_space_before_branch_name = ' ' if self.__git.get_boolean_config_attr('machete.status.extraSpaceBeforeBranchName') else ''
```